Makefile

```diff
@@ -120,10 +120,18 @@ LDFLAGS := $(shell source ./hack/version.sh; version::ldflags)
 ## --------------------------------------
 
 # The number of ginkgo tests to run concurrently
-E2E_GINKGO_PARALLEL=2
+E2E_GINKGO_PARALLEL ?= 2
 
 E2E_ARGS ?=
 
+E2E_GINKGO_FOCUS ?=
+E2E_GINKGO_SKIP ?=
```

Comment:
just to learn this 
what's the FOCUS means ? 100% ratio?

Reply:
Focus means "run only this, skip everything else". So it is for when you want to run only one or a few tests. :slightly_smiling_face: